packages/office-ui-fabric-react/src/components/__snapshots__/Facepile.AddFace.Example.tsx.shot

```diff
@@ -35,18 +59,22 @@ exports[`Component Examples renders Facepile.AddFace.Example.tsx correctly 1`] =
             font-size: 14px;
             font-weight: 400;
             outline: transparent;
-            padding-bottom: 0;
-            padding-left: 16px;
-            padding-right: 16px;
-            padding-top: 0;
+            padding-bottom: 0px;
+            padding-left: 0px;
+            padding-right: 0px;
```

Comment:
These changes here don't look right from a pure sass->getStyles conversion

Reply:
@cliffkoh This seems to match the previous styles: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss#L32 I'm not sure why the test registered a change, but padding: 0 seems to be correct.